pkg/apis/component/v1alpha2/capability_types.go

```diff
@@ -15,34 +14,48 @@ type CapabilitySpec struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
 
-	// Info needed to create a Capability instance from the Capability Catalog
-	// and next to bind the info to a Deployment using the Capability binding/secret
-	Class          string      `json:"class,omitempty"`
-	Plan           string      `json:"plan,omitempty"`
-	ExternalId     string      `json:"externalid,omitempty"`
-	SecretName     string      `json:"secretname,omitempty"`
-	Parameters     []Parameter `json:"parameters,omitempty"`
-	ParametersJSon string
+    /*
+      category: <database>, <logging>,<metrics>
+	  kind: postgres (if category is database)
+	  version: <version of the DB or prometheus or ...> to be installed
+	  secretName: <secret_name_to_be_created> // Is used by kubedb postgres and is optional as some capability provider does not need to create a secret
+	  parameters:
+	     // LIST OF PARAMETERS WILL BE MAPPED TO EACH CATEGORY !
+	    - name: DB_USER // WILL BE USED TO CREATE THE DB SECRET
+	       value: "admin"
+	    - name: DB_PASSWORD  // WILL BE USED TO CREATE THE DB SECRET
+	       value: "admin"
+     */
+	Category       CapabilityCategory  `json:"category"`
+	Kind           CapabilityKind      `json:"kind"`
```

Comment:
We will need to change that name because it's overloaded in the Kubernetes context and might cause confusion.

Reply:
What do you suggest as alternative then ?

Reply:
`Type`?